boards/gardena/sgrm/doc/index.rst

```diff
@@ -0,0 +1,113 @@
+.. _gardena_sgrm:
+
+GARDENA SiM3U radio module
+##########################
+
+Overview
+********
+
+This is a SoM that is used as a radio module by the GARDENA smart gateway (manual_, `FOSS parts`_).
+
+.. figure:: sgrm.webp
+   :align: center
+   :alt: GARDENA SiM3U radio module
+
+
+.. _manual: https://www.gardena.com/tdrdownload//pub000070911/doc000120830
+.. _FOSS parts: https://github.com/husqvarnagroup/smart-garden-gateway-public
+
+Hardware
+********
+
+- Silicon Labs SiM3U167-B-GM_ SoC
+- Silicon Labs Si4467_ transceiver (via SPI)
+- Controls an RGB LED via high drive pins. It's expected to mirror the state of 3 low-drive pins
+  coming from the Linux SoC.
+- UART is connected to the Linux SoC. Usually it's used for PPP, but it can also be used for
+  debugging when PPP is not active.
+
+.. _SiM3U167-B-GM: https://www.silabs.com/mcu/32-bit-microcontrollers/precision32-sim3u1xx/device.SiM3U167-B-GQ?tab=specs
+.. _Si4467: https://www.silabs.com/wireless/proprietary/ezradiopro-sub-ghz-ics/device.si4467?tab=specs
+
+Supported Features
+==================
+
+The board configuration supports the following hardware features:
```

Comment:
*board target, plus put the name as used with west build with \`\` around it

Reply:
Fixed (I think)